docs/tutorials.rst

```diff
@@ -39,11 +39,13 @@ The following tutorials (or sections thereof) explain how to use and extend spec
 * :doc:`How do I set up a batch Bayesian optimization routine?<notebooks/batch_optimization>`
 * :ref:`How do I make a custom acquisition function?<notebooks/failure_ego:Create a custom acquisition function>`
 * :doc:`How do I recover a failed optimization loop?<notebooks/recovering_from_errors>`
+* :doc:`Does Trieste support Ask-Tell optimization?<notebooks/ask_tell_optimization>`
```

Comment:
point of this interface is user-friendly access to BO for users without much experience with BO, so we might want to use something more descriptive here instead of ask-tell?

Reply:
i don't think that's the goal. if anything Ask/Tell is more complex for users to use, because they need to handle the optimization loop themselves.

that said, i am happy to update the description to make it more user-friendly. How about `Does Trieste have interface for external control of the optimization loop?`

Reply:
in some ways its more complex, in others easier... but yes, description is definitely more user-friendly :)

Reply:
ok, changed